packages/plugin-eslint-rules/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@grafana/plugin-eslint-rules",
+  "version": "0.0.4",
+  "description": "An ESlint plugin with Grafana plugin specific rules",
+  "main": "./dist/index.js",
+  "files": [
+    "dist/"
+  ],
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "grafana"
+  ],
+  "homepage": "https://www.npmjs.com/package/@grafana/plugin-eslint-rules",
+  "repository": {
+    "type": "git",
```

Comment:
As it's now in a mono-repo maybe we add the [`directory` property](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#repository) here?

Reply:
Fixed 👍